curriculum/package.json

```diff
@@ -18,7 +18,9 @@
     "lint": "gulp lint",
     "semantic-release": "semantic-release",
     "pretest": "cd ../client && npm run build:workers",
-    "test": "mocha --delay --reporter progress --bail"
+    "pretest:full-output": "npm run pretest",
```

Comment:
I cannot see this ever being used. It is my understanding that `npm run pretest` is the default for `mocha`? So, any call to it would initiate `npm run pretest` directly?

Reply:
That's a `run-script` behavior we are hooking into.

From: https://docs.npmjs.com/misc/scripts.html

> [...]
> Additionally, arbitrary scripts can be executed by running npm run-script <stage>. Pre and post commands with matching names will be run for those as well (e.g. premyscript, myscript, postmyscript). Scripts from dependencies can be run with npm explore <pkg> -- npm run <stage>.